actor/v7action/service_instance_test.go

```diff
@@ -898,4 +899,138 @@ var _ = Describe("Service Instance Actions", func() {
 			})
 		})
 	})
+
+	Describe("CreateManagedServiceInstance", func() {
+		const (
+			fakeServiceOfferingName = "fake-offering-name"
+			fakeServicePlanName     = "fake-plan-name"
+			fakeServiceInstanceName = "fake-service-instance-name"
+			fakeSpaceGUID           = "fake-space-GUID"
+		)
+
+		var (
+			fakeServiceBrokerName = "fake-broker-name"
+			warnings              Warnings
+			err                   error
+			fakeJobURL            = ccv3.JobURL("http://some-cc-api/v3/jobs/job-guid")
+		)
+
+		BeforeEach(func() {
+			fakeCloudControllerClient.GetServicePlansReturns(
+				[]resources.ServicePlan{{GUID: "fake-plan-guid"}},
+				ccv3.Warnings{"plan-warning"},
+				nil,
+			)
+			fakeCloudControllerClient.CreateServiceInstanceReturns(fakeJobURL, ccv3.Warnings{"fake-warning"}, nil)
+			fakeCloudControllerClient.PollJobForStatusReturns(ccv3.Warnings{"job-warning"}, nil)
+		})
+
+		JustBeforeEach(func() {
+			warnings, err = actor.CreateManagedServiceInstance(fakeServiceOfferingName, fakeServicePlanName, fakeServiceInstanceName, fakeServiceBrokerName, fakeSpaceGUID)
+
+		})
+		It("gets the service plan", func() {
+			Expect(fakeCloudControllerClient.GetServicePlansCallCount()).To(Equal(1))
+			query := fakeCloudControllerClient.GetServicePlansArgsForCall(0)
```

Comment:
This tests the query `Values` and not the `Key`.

Reply:
Indeed. I relaxed it because it is using another public method which has tests, so I am just testing that I passed in the right values. This is not mocked though, as they are all methods in the same actor struct, so I'll just add the missing bits